web/pdf_page_view.js

```diff
@@ -548,6 +700,8 @@ class PDFPageView {
 
     const childNodes = div.childNodes,
       zoomLayerNode = (keepZoomLayer && this.zoomLayer) || null,
+      detailScrollLayerNode =
+        (false && keepZoomLayer && this.detailView?.zoomLayer) || null,
```

Comment:
Whops, I should remove this. Also, we should probably enable some lint rule against `false &&` and `true ||`.